tests/test_pipenv.py

```diff
@@ -440,6 +440,26 @@ def test_windows_pinned_pipfile(self):
             assert 'tablib' in p.lockfile['default']
 
 
+    @pytest.mark.e
```

Comment:
This may already be a convention we established but ‘e’ seems really vague and could mean at least two things in this context (editable or the error object). If this is already used, @techalchemy we may want a note about modifying these to a more explicit value.

Reply:
Yep, this was on the test that I copied for this one; my assumption was "editable", so I carried it over.

Reply:
@nateprewitt if there is a convention of marking things with `e` it's possible someone just did that for convenience to isolate a test to run locally, we should be cognizant of that in future PRs and try to get it cleaned up